readme.rst

```diff
@@ -3,63 +3,42 @@ Documentation starter pack
 
-Then go through the following sections to use this starter pack to set up your docs repository.
+Then go through the following sections to use this starter pack to set up your documentation repository.
 
-Set up your documentation repository
-------------------------------------
+Prerequisites
+-------------
 
-You can either create a standalone documentation project based on this repository or include the files from this repository in a dedicated documentation folder in an existing code repository.
+Before you begin, ensure you have the following:
 
-**Note:** We're planning to provide the contents of this repository as an installable package in the future, but currently, you need to copy and update the required files manually.
+* A GitHub repository where you want to host your documentation, cloned to your local machine. The recommended approach is to host the documentation alongside your code in a `docs` folder. But a standalone documentation repository is also an option; in this case, start with an empty repository.
+* Git and Bash installed on your system.
 
-Standalone documentation repository
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Initialise your documentation repository
+---------------------------------------
 
-To create a standalone documentation repository, clone this starter pack
-repository, `update the configuration <#configure-the-documentation>`_, and
-then commit all files to the documentation repository.
+The ``init.sh`` Bash script is used to initialize your repository with the starter pack content. It adds all the files to your repository that are needed to get started with Sphinx documentation.
 
-You don't need to move any files, and you don't need to do any special
-configuration on Read the Docs.
+To use the script, follow these steps:
 
-Here is one way to do this for newly-created fictional docs repository
-``canonical/alpha-docs``:
+* Copy ``init.sh`` to your repository's root directory.
+* Run the script: ``./init.sh``
+* Enter the installation directory when prompted. For standalone repositories, enter ".". For documentation alongside code, enter the folder where your documentation are e.g. "docs".
 
-.. code-block:: none
+This Bash script does the following:
+
+* Clones the starter pack GitHub repository
+* Creates the specified installation directory if necessary
+* Updates working directory paths in workflow files, and updates configuration paths in the ``.readthedocs.yaml`` file
+* Copies and moves contents and ``.github`` files from the starter pack to the installation directory
+* Deletes the cloned repository when it's done
 
-   git clone git@github.com:canonical/sphinx-docs-starter-pack alpha-docs
-   cd alpha-docs
-   rm -rf .git
-   git init
-   git branch -m main
-   UPDATE THE CONFIGURATION AND BUILD THE DOCS
-   git add -A
-   git commit -m "Import sphinx-docs-starter-pack"
-   git remote add upstream git@github.com:canonical/alpha-docs
-   git push -f upstream main
```

Comment:
We're handholding a lot in this PR so losing these steps is going backwards.

Reply:
The PR is not about hand holding at all. It's about convenience. I personally found the steps cumbersome to follow manually and prone to human error. However, nothing in the steps was conceptually confusing. It was just a lot of tedious effort. The purpose of this PR is just to remove that tedious effort.

In the new workflow, these steps don't make sense. IMO anybody setting up a GitHub repository will know how do do basic commit and push operations. Telling them exactly how to do that is like telling someone how to set up their favourite text editor - it's not our job here.

Not only that, but our recommendation is to use an existing repo containing code. So most likely, someone will already have a repo.